apps/desktop/scripts/dev.mjs

```
@@ -26,6 +26,25 @@ const run = (command, args, options = {}) => {
   return child;
 };
 
+const buildWorkspace = (filters) =>
+  new Promise((resolve, reject) => {
+    const child = spawn('pnpm', [...filters, 'build'], {
+      cwd: new URL('../../..', import.meta.url).pathname,
```

ESLint

[error] 'URL' is not defined.

(no-undef)